subiquitycore/tests/mocks.py

```diff
@@ -43,4 +43,9 @@ def make_app(model=None):
     app.opts = mock.Mock()
     app.opts.dry_run = True
     app.scale_factor = 1000
+    app.echo_syslog_id = None
+    app.log_syslog_id = None
+    app.report_start_event = mock.Mock()
+    app.report_finish_event = mock.Mock()
```

Comment:
minor:

I think it's a useful change, thanks! Would you remove those now obsolete instructions (ideally in a separate commit along)?
```
subiquity/server/controllers/tests/test_filesystem.py:        self.app.report_start_event = mock.Mock()
subiquity/server/controllers/tests/test_filesystem.py:        self.app.report_start_event = mock.Mock()
subiquity/server/controllers/tests/test_install.py:        self.controller.app.report_start_event = Mock()
subiquity/server/controllers/tests/test_install.py:        self.controller.app.report_start_event = Mock()
subiquity/server/controllers/tests/test_refresh.py:        self.app.report_start_event = mock.Mock()
subiquity/server/controllers/tests/test_snaplist.py:        self.app.report_start_event = Mock()
```

Reply:
Done! I went ahead and double checked any other instances of these instructions and it looks these were it.